cli/checkpoint/checkpoint.go

```diff
@@ -0,0 +1,40 @@
+package checkpoint
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+
+	"github.com/tarantool/tt/cli/cmdcontext"
+)
+
+// CmdOpts contains flags for managing checkpoint files commands.
+// Used for commands tt cat and tt play, which are checkpoint files commands.
+type CmdOpts struct {
+	To         uint64
+	From       uint64
+	Space      []int
+	Format     string
+	Replica    []int
+	ShowSystem bool
+}
+
+// Cat returns the contents of .snap/.xlog files.
+func Cat(cmdCtx *cmdcontext.CmdCtx) (stdout []byte, err error) {
```

Comment:
Sorry, I don't understand why you are using named return values? AFAIU they can be used for "naked" return and to not declare variables in the code, but you don't use any of them.

Reply:
Simplified to `func Cat(cmdCtx *cmdcontext.CmdCtx) ([]byte, error)`